src/components/VisualPanel/VisualPanel.module.scss

```diff
@@ -0,0 +1,41 @@
+
+.panelWrapper{
+    // width: 300px;
+    // height: 756px;
+    // margin: 30%;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+
+    .title{
+        font-family: Biryani;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 1.3rem;
+        line-height: 32px;
```

Comment:
I suggest using em units here [stack overflow](https://stackoverflow.com/questions/15132671/ems-for-line-height)